### DISCUSSION TITLE
VAR

### DISCUSSION THREAD

**Discussion by MexLemos** [balta-io/forum#130] [Tira Dúvidas] [answered]
O uso do VAR não tira a lógica do "Fortimente tipada"?

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @MexLemos, como vai?

É uma ótima pergunta e na minha percepção não... acredito fortemente que o `var` deixa o código ainda mais limpo.
Se olharmos um exemplo podemos analisar a mudança:
```csharp
string str = "André";
var str = "André";

List<string> nomes = new List<string>();
var nomes = new List<string>();
```

Vale lembrar que **não existe certo ou errado**, se você gosta mais de usar os tipos ao invés de var, tudo bem, OK?

**Reply by MexLemos** [reply]
Obrigado!
